.editorconfig

```diff
@@ -0,0 +1,33 @@
+#http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
```

Comment:
We dont have any js file (well except package.json), so it's safe to delete this.